jurnal-ilmu-dan-teknologi-hasil-ternak.csl

```diff
@@ -0,0 +1,320 @@
+<?xml version="1.0" encoding="utf-8"?>
+<style xmlns="http://purl.org/net/xbiblio/csl" class="in-text" version="1.0" demote-non-dropping-particle="sort-only">
+  <!-- This style was edited with the Visual CSL Editor (http://editor.citationstyles.org/visualEditor/) -->
+  <info>
+    <title>Jurnal Ilmu dan Teknologi Hasil Ternak</title>
+    <title-short>JITEK</title-short>
+    <id>http://www.zotero.org/styles/jurnal-ilmu-dan-teknologi-hasil-ternak</id>
+    <link href="http://www.zotero.org/styles/jurnal-ilmu-dan-teknologi-hasil-ternak" rel="self"/>
+    <link href="http://jitek.ub.ac.id/index.php/jitek/about/submissions#authorGuidelines" rel="documentation"/>
```

Comment:
What style did you use to base this on? Can you please ad a link with rel="template" below the self link?